builtin/plugins/dposv3/dpos.go

```diff
@@ -165,6 +166,10 @@ type (
 	GetRequestBatchTallyRequest = dtypes.GetRequestBatchTallyRequest
 )
 
+type DPOSTotalStaked struct {
```

Comment:
This too doesn't belong here, it should be in the QueryServer.

Reply:
Done